docs/src/content/docs/architecture/testing.md

```
@@ -88,31 +88,31 @@ test("Rook attacks with blockers") {
 
-Example JSON entry:
-```json
-{
-  "fen": "rnbqkbnr/ppp1pppp/8/3pP3/2B5/5Q2/PPPP1PPP/RNB1K1NR w KQkq d6 0 1",
-  "dice": [1],
-  "expectedMoves": ["e5e6", "e5d6", "c2c3", "d2d3", "d2d4", "h2h4"],
-  "title": "En Passant and Path Blockage",
-  "description": "A complex pawn scenario: the pawn on e5 can capture the black d5 pawn en passant (exd6). The c2 pawn's two-square advance is blocked..."
-}
+Example entry:
+```scala
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)